app/models/blacklight_config/search_field_configuration.rb

```diff
@@ -16,7 +16,7 @@ def fields
 
     def normal_fields
       %w{title description creator lcsubject date institution}.map do |field|
-        SearchField.new(field)
+        SearchField.new(field, :qf => Solrizer.solr_name(field, :stored_searchable))
```

Comment:
Instead of adding the option here, go into the SearchField model and change the default to :stored_searchable